### ISSUE TITLE
Corvette - Starwave

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Skywanderers-Pathfinder-Group-Build#1]
Got a blueprint ready to send your way!
![20200124221957_1](https://user-images.githubusercontent.com/33041786/73116187-af78df80-3ef7-11ea-9c68-fe51aebd7185.jpg)

This version has had the interior removed and resized for 3m high ceilings as requested. It's all function; no fancy interior, and all the brick detailing on the exterior has been removed (apart from 2 stickers to make the paint look less derpy). Obviously, a release-ready version would look nicer, but this is purely for testing performance & balance.

[Starwave Corvette - 1st Testing Version.zip](https://github.com/tsunamayo/Skywanderers/files/4111614/Starwave.Corvette.-.1st.Testing.Version.zip)

**Comment by tsunamayo** [comment]
Great ship! I like your style, it is very cohesite, it use well the stretch tech without overdoing it.
I see what you means for the 3m ceiling, it looks way too big indeed, the problem being the door I think. I think we need to use 2m doors, either 2mx1 for small room or 2mx2m for larger one, 3m gives an oversize feel. Then it is always possible to add some gribbling to the ceiling.
I need to check more 2m high ship to get a better idea for it, I will get back to you.

But you can continue on detailing (light also is super important), in any case even if I allow 2m, 3m ship would still be okay.

Size is also perfect, and the interior layout makes of lot of sense, we will be able to tweak systems moving forward. I also see for the fps drop linked to turret, I will have a look at it asap.

Nicely done, thanks a lot!

**Comment by tsunamayo** [comment]
Ah also side note, if you could work on some landing gear (or do you want it to land like that?), and an access ramp / elevator, as it could be a tricky point.
Thanks!

**Comment by Dwarf-LordPangolin** [comment]
Thank you! I'm really glad you like it. I've switched the doors to 2m, and have started on detailing/lighting the interior.

BTW: thank you for clarifying that greebling won't deduct from ceiling height. 3m with greebling looks a lot better than without; it's better than both a plain 3m ceiling, and a 2m ceiling.

And yes, it will definitely have landing gear and a ramp! It had them originally, but they were designed to look pretty, and weren't performance friendly, so they needed to be redone.

Quick preview of the interior so far; too much detail, too little, or just right?
![20200129022948_1](https://user-images.githubusercontent.com/33041786/73341816-03aaf900-4243-11ea-9f23-d351ce01412d.jpg)

**Comment by Dwarf-LordPangolin** [comment]
OK! The detailing is all finished; let me know if this is too much detail. Landing gear and a ramp/airlock have been added as well. As with the others, turrets are configured, just not hooked up to the yoke. Let me know what it needs next! :)

[Starwave Corvette - v2.zip](https://github.com/tsunamayo/Skywanderers-Pathfinder-Group-Build/files/4143509/Starwave.Corvette.-.v2.zip)

**Comment by tsunamayo** [comment]
Okay interior is super clean, and really perfect for my current need. I have a lot of trouble with the landing gear, which is too tight for the collision system. Also the turrets are not looking at the right direction, (I did the link myself), it might be linked to the fact that you linked several turrets to the same core, it was not made to work like that (indeed I need to prevent it...). I keep you posted!

**Comment by Dwarf-LordPangolin** [comment]
Yeah; I see that the moving parts aren't functioning properly anymore. I'll do a redesign of the ramp & landing gear so they work again, and also set the turrets up with individual turret cores.

**Comment by Dwarf-LordPangolin** [comment]
[Starwave Corvette - v3.zip](https://github.com/tsunamayo/Skywanderers-Pathfinder-Group-Build/files/4533804/Starwave.Corvette.-.v3.zip)

![20200425134859_1](https://user-images.githubusercontent.com/33041786/80288192-eaa7cc00-86fb-11ea-8681-404e7f0a8370.jpg)

The Starwave has been updated! Among other changes, I switched the small turrets to beams to give it more of that "weapon cocktail" thing you were talking about. I also changed their color to blue to see how that looks; I was originally going to color weapons based on their power, with red for weaker ones and blue for more powerful ones, but I'm starting to think maybe it would look better to give the faction as a whole a unified weapon color. Since the blue engine glow goes well with the paint scheme, I'll probably do the weapons in the same color.

**Comment by tsunamayo** [comment]
Yes I concur, I also like to stick to a single glow accent color that is complementary to the color scheme. For my gray-red shuttle it is green, once I do a gray-blue it will be orange, ect. 
Will check it out!
